packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -83,7 +83,10 @@ export const MenuItemStyles = css`
     }
 
     .start,
-    .end,
+    .end {
+        display: flex;
+    }
```

Comment:
Are these two the only places we've baked in the start/end to 16px?

Reply:
After another check, I missed text-field, tree-item, and breadcrumb